src/libraries/Microsoft.PowerFx.Interpreter/Functions/StandardErrorHandling.cs

```diff
@@ -271,22 +289,33 @@ private static AsyncFunctionPtr StandardErrorHandlingTabularOverload<TScalar>(st
             return StandardSingleColumnTable<T>(async (runner, context, irContext, args) => targetFunction(irContext, args.OfType<T>().ToArray()));
         }
 
-        private static (int maxTableSize, bool emptyTablePresent) AnalyzeTableArguments(FormulaValue[] args)
+        private static (int maxTableSize, int minTableSize, bool emptyTablePresent, bool blankTablePresent) AnalyzeTableArguments(FormulaValue[] args)
         {
             var maxTableSize = 0;
             var emptyTablePresent = false;
+            var minTableSize = int.MaxValue;
+            var blankTablePresent = false;
 
             foreach (var arg in args)
             {
                 if (arg is TableValue tv)
                 {
                     var tableSize = tv.Rows.Count();
                     maxTableSize = Math.Max(maxTableSize, tableSize);
+
+                    // Empty tables are not considered for count.
+                    minTableSize = tableSize != 0 ? Math.Min(minTableSize, tableSize) : minTableSize;
+
                     emptyTablePresent |= tableSize == 0;
                 }
+
+                if (arg is BlankValue bv)
+                {
+                    blankTablePresent = true;
```

Comment:
>blankTablePresent [](http://example.com/codeflow?start=20&length=17)

This is only true if `bv.IRContext.ResultType._type.IsTable`; otherwise it may be a blank scalar value

Reply:
Thanks for pointing that out.